pyrato/edc.py

```diff
@@ -67,17 +67,20 @@ def schroeder_integration(room_impulse_response, is_energy=False):
     The result is the energy decay curve for the given room impulse response
     after [#]_.
 
-    .. math:
+    .. math::
 
-        \langle e^2(t) \rangle = N\cdot \int_{t}^{\infty} h^2(\tau)
+        \langle e^2(t) \rangle = \int_{t}^{\infty} h^2(\tau)
         \mathrm{d} \tau
 
     Parameters
     ----------
-    room_impulse_response : pyfar.Signal
-        Room impulse response as array
+    room_impulse_response : pyfar.Signal, pyfar.TimeData
+        Room impulse response as pyfar audio object.
```

Comment:
```suggestion
        The room impulse response.
```
The type is redundant.